dockerfiles/is/change_ownership.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# ------------------------------------------------------------------------
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# ------------------------------------------------------------------------
+
+set -e
+
+echo 'going to change ownership of <IS_HOME>/repository/data directory: '
+echo "user: ${USER}"
+echo "user home: ${USER_HOME}"
+echo "carbon server: ${WSO2_SERVER}-${WSO2_SERVER_VERSION}"
+
+/bin/chown -R ${USER} ${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}/repository/deployment/server/
```

Comment:
We might need to remove this file. Once proper security rules are applied using Pod Security Policies at the Kubernetes cluster level, this script would not work. Persistent volume permissions might need to be granted using the operating system user group.